runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java

```diff
@@ -117,11 +120,20 @@ public void onGameTick(GameTick gameTick)
 			Duration statTimeout = Duration.ofMinutes(config.statTimeout());
 			Duration sinceCut = Duration.between(fermentTimerSession.getLastWineMakingAction(), Instant.now());
 
-			if (sinceCut.compareTo(statTimeout) >= 0)
+			if (client.getLocalPlayer().getAnimation() == COOKING_WINE && config.fermentTimer())
+			{
+				fermentTimerSession.updateLastWineMakingAction();
+			}
+			else if (sinceCut.compareTo(statTimeout) >= 0)
 			{
 				fermentTimerSession = null;
 			}
 		}
+		else if (config.fermentTimer() && client.getLocalPlayer().getAnimation() == COOKING_WINE)
+		{
+			fermentTimerSession = new FermentTimerSession();
```

Comment:
This should just use the animation change event probably.

Reply:
You mean remove the config.fermentTimer() condition, correct? Or is there a specific animation event that I'm unaware of?

Reply:
Just found the class, nvm.

Reply:
Fixed.